app/src/main/java/bitcoin/wallet/core/managers/PreferencesManager.kt

```diff
@@ -88,6 +90,25 @@ class PreferencesManager(private val encryptionManager: IEncryptionManager) : IL
         return if (json?.isBlank() == true) defaultCurrency else gson.fromJson<Currency>(json, Currency::class.java)
     }
 
+    override fun setUnlockAttemptsNumber(attemptsNumber: Int) {
+        App.preferences.edit().putInt(UNLOCK_PIN_ATTEMPTS_LEFT, attemptsNumber).apply()
+    }
+
+    override fun getUnlockAttemptsNumber(): Int {
+        return App.preferences.getInt(UNLOCK_PIN_ATTEMPTS_LEFT, 5)
```

Comment:
Would be better to make a config constant like pin_unlock_max_attempts = 5. It can be used in UnlockInteractor too.